CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Change Log
 
+# [1.2.0 (June 29th, 2021)](https://github.com/eclipse/eclipse.jdt.ls/milestone/92?closed=1)
+ * enhancement - Add a preference to control the severity of not covered maven plugin execution. See [#1770](https://github.com/eclipse/eclipse.jdt.ls/pull/1770).
+ * enhancement - Allow folding `static` blocs. See [#1777](https://github.com/eclipse/eclipse.jdt.ls/issues/1777).
+ * enhancement - Add deprecated property to CompletionItem and SymbolInformation. See [#695](https://github.com/eclipse/eclipse.jdt.ls/issues/695).
+ * enhancement - deprecated is not populated in textDocument/documentSymbol. See [#1301](https://github.com/eclipse/eclipse.jdt.ls/issues/1301).
+ * bug fix - Cannot make a static reference to the non-static type T. See [#1781](https://github.com/eclipse/eclipse.jdt.ls/issues/1781).
+ * bug fix - Standard language server should also be able to exit on shutdown. See [#1808](https://github.com/eclipse/eclipse.jdt.ls/pull/1808).
+ * bug fix - File contents would be strange when renaming java file name (with lombok). See [#1775](https://github.com/eclipse/eclipse.jdt.ls/issues/1775).
+ * bug fix - java.project.sourcePaths doesn't refresh diagnostics. See [#1769](https://github.com/eclipse/eclipse.jdt.ls/issues/1769).
+ * bug fix - wrong status in 'language/progressReport' notification when processing call hierarchy requests. See [#1722](https://github.com/eclipse/eclipse.jdt.ls/issues/1722).
+ * build - Update target platform to 2021-06 RC1. See [#1783](https://github.com/eclipse/eclipse.jdt.ls/issues/1783).
```

Comment:
The latest target platform we used is 4.21 I-build.
https://github.com/eclipse/eclipse.jdt.ls/pull/1804